dependencies.properties

```diff
@@ -30,6 +30,30 @@ jna-platform.key=CHK@G0F322jl0AxJqk-yfuRes5vPcpR~eb3pXjckqjr0qmU,mEDDjQHrPDt3Kt9
 jna-platform.size=2327597
 jna-platform.type=CLASSPATH
 jna-platform.order=3
+pebble.version=3.1.5
+pebble.filename=pebble-3.1.5.jar
+pebble.filename-regex=pebble-*.jar
+pebble.key=CHK@y~p8HMUVXmVgfSnrmUyu2UNXMO9uMDHS5nwo2YuOKvw,yzwLFP0GXa8RjwRpicQCPFKNggDXLkTQKH8nISe0qUY,AAMC--8/pebble-3.1.5.jar
+pebble.size=318169
+pebble.sha256=85e77f9fd64c0a1f85569db8f95c1fb8e6ef8b296f4d6206440dc6306140c1a1
```

Comment:
```
steve@steve-ubuntu ~/Downloads> sha256sum pebble-3.1.5.jar 
d253a6dde59e138698aaaaee546461d2f1f6c8bd2aa38ecdd347df17cf90d6f0  pebble-3.1.5.jar
```

Hm, I'm seeing a different hash. I got mine from https://mvnrepository.com/artifact/io.pebbletemplates/pebble/3.1.5 - did you get yours from the build fetching it? Do you know where from?

Reply:
Arg, that would have been an awful failure mode. Checking the sha256sum of the jar, I also get d253a6dde59e138698aaaaee546461d2f1f6c8bd2aa38ecdd347df17cf90d6f0 — but there is another entry with the same hash:

`wrapper-update-mac.files.libwrapper-macosx-universal-64.sha256=85e77f9fd64c0a1f85569db8f95c1fb8e6ef8b296f4d6206440dc6306140c1a1`

which is where I copied the keys from. So  this is a copy-paste error.

Thank you for checking! :heart:

Reply:
Oh! I didn't think to check for that. It seems like it'd make sense to test that the dependencies are properly fetched, then! Would that be doable with a testing update USK?

Reply:
It should be fixed now.